packages/discovery-react-components/src/components/CIDocument/components/CIDocument/__tests__/CIDocument.spec.tsx

```diff
@@ -209,11 +209,11 @@ describe('<CIDocument />', () => {
     it('displays the correct party information on the details pane', async () => {
       // Expects metadata pane to be hidden until selected
       const metadataPane = getByTestId('metadata-pane').parentElement!;
-      expect(metadataPane.getAttribute('aria-hidden')).toEqual('true');
+      expect(metadataPane).toHaveProperty('hidden', true);
```

Comment:
in order to use `expect(await screen.findByRole('tab', { name: 'Document', selected: false })).toBeVisible();` we need to upgrade `@testing-library/react` from `9` to `>=10`, which causes a bunch of other test failures